scripts/dealgenie_scorer.py

```
@@ -81,370 +245,914 @@ def get_default_config(self) -> Dict[str, Any]:
+        df_scored = scorer.score_properties(df, batch_size=500)
+        
+        # Step 6: Export results with validation
+        logger.info("\n6. Exporting results...")
+        output_file = 'sample_data/scored_zimas_properties.csv'
+        
+        try:
+            # Ensure output directory exists
+            Path(output_file).parent.mkdir(exist_ok=True)
+            df_scored.to_csv(output_file, index=False)
+            logger.info(f"   ✓ Results exported to {output_file}")
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)